packages/geo/src/lib/search/search-results/search-results-add-button.component.ts

```diff
@@ -350,6 +377,9 @@ export class SearchResultAddButtonComponent implements OnInit, OnDestroy{
                 }
               }
             },
+            style: function(feature) {
```

Comment:
@aziz-access . Please, do not create a stylefunction unless you really need it.  ==>  style: style

Reply:
@aziz-access I've done the fix.

Reply:
thank you